subsystem/src/main/java/org/wildfly/extension/grpc/GrpcExtension.java

```diff
@@ -61,4 +67,10 @@ public void initializeParsers(ExtensionParsingContext extensionParsingContext) {
         extensionParsingContext.setSubsystemXmlMapping(SUBSYSTEM_NAME, GrpcSubsystemParser_1_0.NAMESPACE,
                 CURRENT_PARSER);
     }
+
+    private static void registerAttributes(ManagementResourceRegistration registration) {
+        for (AttributeDefinition definition : GrpcAttribute.ATTRIBUTES) {
+            registration.registerReadWriteAttribute(definition, null, new GrpcParamHandler(definition));
```

Comment:
We should just use the `ReloadRequiredWriteAttributeHandler`. Unless the service itself can be restarted.